### DISCUSSION TITLE
blood samples item checking the wrong tag

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14460] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

It should prevent gaining condition from already sampled targets by applying a tag, but it uses the wrong tag, so you can sample the same target multiple times

sampletaken
bloodsampletaken

![image](https://github.com/user-attachments/assets/27aebfa8-9633-42fd-8986-02b4d427b2bf)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
To me it seems like this is some leftover from some earlier approach to preventing "duplicate samples". The tag should most likely not be "bloodsamples" - that won't work either because there's no statuseffect with that tag.

Since the event doesn't count the "duplicate samples", to me this doesn't seem important to address.
